vxwhatsapp/consumer.py

```diff
@@ -93,9 +93,13 @@ async def process_message(self, message: IncomingMessage):
             await message.reject(requeue=False)
             return
 
-        async with message.process(requeue=True):
+        async with message.process():
             logger.debug(f"Processing outbound message {msg}")
-            await self.submit_message(msg)
+            try:
+                await self.submit_message(msg)
+            except aiohttp.ClientResponseError as e:
+                if e.status > 499:
+                    await message.reject(requeue=True)
```

Comment:
This is quite a big change. Before any exception would get retried, now only ClientResponseErrors will be.

I'm wondering if we should have another `except Exception:`, or if we're just okay with those other errors boiling up.

One thing I do think we need here, is if the status code is <=499, then we need to log that exception, so that it gets picked up with sentry, otherwise that message is silently getting thrown away.

Reply:
Good idea!